README.md

```diff
@@ -1,87 +1,97 @@
 # OpenWeave MFGN Examples
 
-Example applications for MFGN partners showing the use of OpenWeave for a variety of
-device types and hardware platforms.
+Example applications for MFGN partners showing the use of OpenWeave for
+a variety of device types and hardware platforms.
 
-[OpenWeave](https://github.com/openweave/openweave-core/) is the open source release of
-Nest's Weave technology, an application layer framework that provides a secure,
-reliable communications backbone for Nest's products.
+[OpenWeave](https://github.com/openweave/openweave-core/) is the open
+source release of Nest's Weave technology, an application layer
+framework that provides a secure, reliable communications backbone for
+Nest's products.
 
-<hr>
+---
 
 <a name="intro"></a>
```

Comment:
I was going to remove these, as markdown auto-generates anchors for each section heading.  I didn't because something might be using them.  These also create stable references, where the Markdown ones change if you change the section heading.